radix-common/src/data/manifest/model/manifest_resource_assertion.rs

```diff
@@ -753,6 +957,13 @@ impl AllowedIds {
         }
     }
 
+    pub fn has_constraints(&self) -> bool {
```

Comment:
I think this should be removed, as I think not used now? And also misleading? As even an empty `allowlist` is a constraint. (in fact, a very tight one!)